addons/web/static/src/scss/bootstrap_review.scss

```diff
@@ -22,7 +22,7 @@
     // to a BS3-like one: only the header and footer are colored and the body
     // will use the color of a default card background with a light opacity.
     // Limitation: bg-* utilities cannot be used on card-body elements anymore.
-    @include o-bg-color(rgba($card-bg, $o-card-body-bg-opacity));
+    @include o-bg-color(rgba($card-bg, $o-card-body-bg-opacity), $with-muted: false);
```

Comment:
Not sure about this anymore, it seems to add bugs rather than solving them... Can you remind me? Because use a text-muted in the body of a `.card.bg-secondary`... and it will not be visible with this :/

Reply:
Something probably changed meanwhile (?), If I remember well using `text-muted` on some cards was problematic... we come out with this workaround, but you were not happy so you may have found a better solution meanwhile.
Sorry I don't remember more ^^

Btw I can't drop this, is not really needed for this task 👍

Reply:
I don't remember either... I'd say we drop this then, I tested and it breaks the colored cards to only *very slightly* improving the normal cards ;)